src/calendar/assets/calendar/skins/sam/calendar-skin.css

```diff
@@ -1,7 +1,3 @@
 .yui3-skin-sam .yui3-calendar-day-highlighted {
-    background-color: #DCDEF5;
-}
-
-.yui3-skin-sam .yui3-calendar-day-selected.yui3-calendar-day-highlighted {
-    background-color: #758FBB;
-}
+    background-color: #DCDEF5 !important;
+}
```

Comment:
Hey @ethib137, the main objective of this change was to bring this style to the selected days. Because they did not undergo any style change when highlighted. 

The style I overwrote was this:

![screenshot-1726667177](https://github.com/user-attachments/assets/a4b3095d-9c39-49af-bb1c-c5afcec1a707)

I tried to do this in other more elegant ways, but to no avail, let me know if you have any advice on this.